Source/SwiftLintFramework/Models/Version.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Version.swift
+//  SwiftLint
+//
+//  Created by HirayamaYuya on 2016/09/16.
+//  Copyright © 2016年 Realm. All rights reserved.
+//
+
+import Foundation
+
+public struct Version: CustomStringConvertible {
+
+    public static var current: Version {
+        get {
+            guard let version = NSBundle(identifier: "io.realm.SwiftLintFramework")?
+                .objectForInfoDictionaryKey("CFBundleShortVersionString") as? String else {
+                    fatalError()
+            }
+
+            return Version(versionString: version)
+        }
+    }
+
+    var major: Int
+    var minor: Int
+    var patch: Int
+
+    init(major: Int, minor: Int, patch: Int) {
+        self.major = major
+        self.minor = minor
+        self.patch = patch
+    }
+
+    init(versionString: String) {
+        let numbers = versionString.componentsSeparatedByString(".").map {
+            return Int($0)!
```

Comment:
You can crash hard here. Consider failing the initializer or throwing an error when not parsable instead.